main/src/library/RedBlackSet.flix

```diff
@@ -0,0 +1,296 @@
+///
+/// An experimental RedBlackSet type represented internally as a red-black tree.
+///
+///
+enum RedBlackSet[t] {
+    case RedBlackSet(RedBlackTree[t])
+}
+
+namespace RedBlackSet {
+
+    ///
+    /// Returns the size of `xs`.
+    ///
+    pub def size(xs: RedBlackSet[a]): Int32 =
+        let RedBlackSet(es) = xs;
+            RedBlackTree.size(es)
```

Comment:
I know existing code is formatted like this, but it should actually be:

```
    pub def size(xs: RedBlackSet[a]): Int32 =
        let RedBlackSet(es) = xs;
        RedBlackTree.size(es)
```
without the extra indent following the let.